ipatests/test_integration/test_commands.py

```diff
@@ -1598,6 +1598,57 @@ def test_ipa_cacert_manage_prune(self):
 
         assert isrgrootx1_nick in result
 
+    def test_ipa_systemd_journal(self):
+        """
+        This testcase checks that administrative user credentials
+        is not leaked to journald log
+        """
+        tasks.kinit_admin(self.master)
+        tasks.kinit_admin(self.replicas[0])
+        tasks.kinit_admin(self.clients[0])
+        cmds = [
+            ['/usr/sbin/ipa-adtrust-install', '-a',
+             self.master.config.admin_password, '-U'],
+            ['/usr/sbin/ipa-replica-manage', 'del',
+             f"dummyhost.{self.master.domain.name}", '-p',
+             self.master.config.dirman_password],
+            ['/usr/sbin/ipa-csreplica-manage', 'del',
+             f"dummyhost.{self.master.domain.name}", '-p',
+             self.master.config.dirman_password],
+            ['/usr/sbin/ipa-kra-install', '-p',
+             self.master.config.dirman_password, '-U'],
+            ['/usr/sbin/ipa-server-certinstall', '-k', '--pin',
+             self.master.config.dirman_password, '-p', 
```

Comment:
This line triggers a lint failure:
ipatests/test_integration/test_commands.py:1621: [C0303(trailing-whitespace), ] Trailing whitespace)